.github/workflows/assets.yml

```diff
@@ -113,11 +115,11 @@ jobs:
           # Create SHA256 checksum for rootfs.img
           sha256sum "assets/rootfs.img" | awk '{ print $1 }' > "assets/rootfs.img.sha256"
```

Comment:
I thought the sha256 failed? Did I misread the error?

Reply:
The failure was not related to the sha. The mv command was failing as the variable name was incorrect.

Reply:
@eriknordmark iIf the changes look good, I'd like to merge this, backport it to 13.6, and create a tag to test the workflow.